### DISCUSSION TITLE
[预览]v2.0.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2495] [分享 / Share]
- `动态过滤器` 支持屏蔽新版的话题. (#2479)
- 修复一些图标在新版动态首页尺寸错误.
- 新增插件 `快捷键扩展 - 无动作`, 将按键绑定到这个动作上可以阻止原有的事件处理. (#2474)
- 夜间模式更新. (PR #2491 by [JLoeve](https://github.com/LonelySteve))
- 支持自动破坏 `spm_id`. (#2477)
- 修复 `自动更新器` 下载失败时弹出选择文件窗口. (#2450)
- 修复侧栏在特殊直播间被壁纸遮挡. (#2484)
- GitHub CDN 更换 Raw 直链, 减少跳转. (#2409)
- 修复 `在线仓库` 在某些分辨率下文字模糊. (#2475)
- 设置面板里的组件详情改为固定位置, 这样可以支持滚动和更好的动画效果, 防止长内容超出页面.
- 因 API 失效, `查看封面` 组件不再支持在直播间中使用.
- `直播勋章快速更换` 弹窗支持滚动, 可以显示更多数量的勋章, 并按等级降序排列. (#2448)
- 修复 `直播勋章快速更换` 中的勋章大于 20 级后样式错误. (#2448)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.1-preview'>[预览]v2.0.1</a>.</em>

**Comment by BunnySmasher** [comment]
大佬压缩包里面文件还是v2.0.0的
![QQ截图20211108210635](https://user-images.githubusercontent.com/34413382/140747340-4df61391-3ba3-44a9-96d8-22046bc67fd5.png)
![QQ截图20211108210845](https://user-images.githubusercontent.com/34413382/140747439-121a1baf-a641-4fdc-948d-331dd575567d.png)

**Reply by the1812** [reply]
安装是在 [README](https://github.com/the1812/Bilibili-Evolved/tree/preview#%E5%AE%89%E8%A3%85) 那边, 从来不是下压缩包来装的

**Reply by BunnySmasher** [reply]
好吧,我一直这么按的,谢谢大佬

**Comment by Geeyun-JY3** [comment]
对 v2 一直有个问题，如何确定组件的版本，方便判断本机上已安装的是否已修复 bug

**Reply by the1812** [reply]
可能考虑像本体版本一样塞个 commit hash 进去

**Comment by thouser8** [comment]
请问下作者大哥，2.0.0怎么更新到2.0.1呢？2.0.0版本没法导出设置，重新安装2.0.1的话需要手动逐个安装功能

**Reply by the1812** [reply]
油猴里检查更新, 或者直接下 2.0.1 覆盖升级

**Comment by thouser8** [comment]
使用最新版v2.0.1（Bilibili Evolved (Preview) v2.0.1）安装了插件后设置面板里没有显示
![DJFS3 F2Z1N3M0K4AM~SV@O](https://user-images.githubusercontent.com/83090571/141672946-7ecf8f4a-f17f-46f6-bf8e-9db7dbe485f9.png)
![9S8CDESADC%A$65T0{FV~5V](https://user-images.githubusercontent.com/83090571/141672970-c8451e09-db3b-4770-857d-ce3b420f39fa.png)

而使用无法更新的旧版本(Bilibili Evolved (v2) v2.0.0)能正常显示
![K05@AP`D`6A({RX33L()%YL](https://user-images.githubusercontent.com/83090571/141672991-f9a85c93-6296-4c2f-96e7-a44a883c37e7.png)
![`PLAOP}92}RL1`ECWDW W1B](https://user-images.githubusercontent.com/83090571/141672999-bb5da0f5-118f-4dbb-ba40-95f1e9bc2204.png)
但无法更新版本，同样无法导出设置（点击导出会传送到 t.bilibili.comhttps开头的网址）

**Reply by the1812** [reply]
1. 你把插件装到组件里了, 卸了重新来
2. 旧版不能在动态页里做任何导出动作

**Comment by BlueFunny19** [comment]
这一版真的精致了好多，和上一个版本简直天差地别了

**Reply by soulpowerhang** [reply]
![image](https://user-images.githubusercontent.com/94303921/141686166-ed344634-dd5a-4ce6-ad08-e77c224ca71d.png)
请问一下 为什么我的设置按钮找不到了

**Reply by Ranlajetech** [reply]
有一个自动隐藏的功能，是默认关闭的，你如果在v1里打开了自动隐藏，升上来会继承原来的配置，你原来的按钮放在什么位置，用鼠标过一下那个地方可能会出来（已知问题：按钮置右和细滚动条这两个功能有冲突，自动隐藏了按钮就再也出不来了，因为判定区域重合）

**Reply by soulpowerhang** [reply]
ok 我重置了一下 现在出来了

**Reply by the1812** [reply]
> 已知问题：按钮置右和细滚动条这两个功能有冲突，自动隐藏了按钮就再也出不来了，因为判定区域重合

不是出不来, 其实是在滚动条左边 1px 的位置能触发, 当然这已经违背了边缘触发的初衷了, 体验很差. 所以描述里写的是**不建议**置右和自动隐藏一起用.

**Comment by Loveshy1** [comment]
请问以前的简洁版主页现在怎么开启，我只找到简化主页选项，但没找到简洁版主页

**Reply by Geeyun-JY3** [reply]
等：#2448
> 暂不支持 v1 的 `简化首页` 功能, 在 v2 中 `简化首页` 等于 v1 的 `首页过滤` 功能. v1 的 `简化首页` 由于代码设计过于糟糕, 很难再进一步扩充功能, 我计划在 v2 发布后重写这个功能, 届时会比 v1 更加美观和强大. 如果你非常需要这个功能, 请暂时不要升级到 v2.

**Reply by Loveshy1** [reply]
感谢回复，十分期待你们的更新
